engine/src/test/java/org/hibernate/validator/test/predefinedscope/PredefinedScopeValidatorFactoryTest.java

```diff
@@ -182,24 +182,6 @@ public void testValidOnUnknownBean() {
 		assertNoViolations( violations );
 	}
 
-	@Test(expectedExceptions = ValidationException.class, expectedExceptionsMessageRegExp = "HV000250:.*")
-	public void testUninitializedLocale() {
```

Comment:
Yeah, I got rid of this test a bit too quickly. It was not passing anymore because of the default locale always being added. I changed it a bit to be able to trigger the error.